src/features/user_profile/components/SettingsForm/constants.tx.ts

```diff
@@ -0,0 +1,2 @@
+export const MAX_HEIGHT = '250px';
```

Comment:
MAX_HEIGHT and MIN_HEIGHT of what? So far it's only used for Bio textarea, so `MIN(MAX)_BIO_HEIGHT` would be more obvious